docs/classes.rst

```diff
@@ -64,9 +64,16 @@ Step
 
         The :class:`Background` this step belongs to (if inside a background).
 
+    .. attribute:: test
+
+        The instance of :class:`unittest.TestCase` running the current test,
+        or None if not currently in a test (e.g. in a
+        :func:`~aloe.before.each_feature` callback).
```

Comment:
Don't think that's needed - the only way to get to steps in this case is through the feature, which has the reference itself.